### PR TITLE
Chore/RB-33 Implement audit changes to Dockerfile

### PR DESCRIPTION
**Pull request by NilsonBertolaAB**
# Description
Implement audit changes to Dockerfile
- Fix installed packages versions
- Remove cache after instalation

# Task
[RB-33](https://appliedblockchain.atlassian.net/browse/RB-33)

### CODE REVIEW

File: services/app/Dockerfile [hunk 1]
```diff
@@ -6,7 +6,8 @@ RUN npm install && npm run build:proto && npm run build
 
 # node:16.16.0-alpine3.16
 FROM node@sha256:d39ab4712a8395d0b399dea44d9cb8b34ac942411b6a380449ebdb9d321136a3 as prod
-RUN apk add dumb-init
+RUN apk add --no-cache dumb-init=1.2.5-r1
+RUN rm -rf /var/cache/apk/*
```
Comment (campbell-codes) [inline services/app/Dockerfile:10]: Is this needed if cache is disabled anyway?

File: services/vrf/Dockerfile [hunk 2]
```diff
@@ -2,7 +2,8 @@
 FROM node@sha256:d39ab4712a8395d0b399dea44d9cb8b34ac942411b6a380449ebdb9d321136a3 as builder
 WORKDIR /usr/src/app
 COPY . /usr/src/app
-RUN apk add --no-cache --virtual .gyp python3 make gcc py3-pip g++
+RUN apk add --no-cache --virtual .gyp python3=3.10.5-r0 make=4.3-r0 gcc=11.2.1_git20220219-r2 py3-pip=22.1.1-r0 g++=11.2.1_git20220219-r2
+RUN rm -rf /var/cache/apk/*
```
Comment (campbell-codes) [inline services/vrf/Dockerfile:6]: Same

File: services/vrf/Dockerfile [hunk 3]
```diff
@@ -12,7 +13,8 @@ RUN npm install &&\
 
 # node:16.16.0-alpine3.16
 FROM node@sha256:d39ab4712a8395d0b399dea44d9cb8b34ac942411b6a380449ebdb9d321136a3 as prod
-RUN apk update && apk add dumb-init
+RUN apk add --no-cache dumb-init=1.2.5-r1
+RUN rm -rf /var/cache/apk/*
```
Comment (campbell-codes) [inline services/vrf/Dockerfile:17]: Same

**Review by campbell-codes — CHANGES_REQUESTED** [review]
If we need the rm then it's good I'm happy to approve just not sure we do

**Review by seromenho — APPROVED** [review]
Good stuff 👌

**Review by campbell-codes — APPROVED** [review]
Looks good

**Pull request merged**